pgmpy/base/DAG.py

```diff
@@ -85,6 +89,38 @@ def __init__(self, ebunch=None, latents=set()):
             out_str += "".join([f"({u},{v}) " for (u, v) in cycles])
             raise ValueError(out_str)
 
+            ebunch, latents, err_corr, err_var = parse_lavaan(kwargs["lavaan_str"])
```

Comment:
Is this needed as we are already parsing in line 77?

Reply:
Thanks for the catch. I will delete the line. It was accidentally there. :-0

Reply:
Fixed it 573c6208a4f97892560d4ba6e6894267de903ebd